lib/logstash/outputs/scalyr.rb

```diff
@@ -60,6 +60,22 @@ class LogStash::Outputs::Scalyr < LogStash::Outputs::Base
   # (Warning: events with an existing 'logfile' field, it will be overwritten)
   config :logfile_field, :validate => :string, :default => 'logfile'
 
+  # Record field which includes the value for the "severity" field. severity is a special field which tells
+  # Scalyr severity / log level for a particulat event. This field is a top level event field and not
+  # event attribute field. Actual field value must be an integer and is mapped to different severity /
+  # log level on DataSet server side as shown below:
+  #
+  # - 0 -> finest
+  # - 1 -> trace
+  # - 2 -> debut
+  # - 3 -> info
+  # - 4 -> warning
+  # - 5 -> error
+  # - 6 -> fatal / emergency / critical
+  #
+  # By default, if Event contains no severity field, default value of 3 (info) will be used.
```

Comment:
Just to clarify for me: 
> default value of 3 (info) will be used.

Event - DataSet event, the default value is set on server side?

Reply:
Correct, if not provided as part of the Event.sev field, server-side sets the value to 3 (info).